src/utils.cpp

```diff
@@ -178,6 +179,54 @@ StdioSilencer::~StdioSilencer()
   }
 }
 
+KConfig::KConfig()
+{
+  std::vector<std::string> config_locs;
+
+  // Try to get the config from BPFTRACE_KCONFIG_TEST env
+  // If not set, use the set of default locations
+  const char *path_env = std::getenv("BPFTRACE_KCONFIG_TEST");
+  if (path_env)
+    config_locs = { std::string(path_env) };
+  else
+  {
+    struct utsname utsname;
+    if (uname(&utsname) < 0)
+      return;
+    config_locs = {
+      "/boot/config-" + std::string(utsname.release),
+      "/proc/config.gz",
```

Comment:
Maybe order procfs option first. The /boot/.. configs can suffer from skew when installing new kernels.